core/src/main/java/com/kin/ecosystem/core/data/order/OrderRepository.java

```diff
@@ -162,6 +163,13 @@ public void onFailure(ApiException e) {
 		});
 	}
 
+	@Override
+	public OpenOrder createOutgoingTransferOrderSync(@NonNull final OutgoingTransferRequest request) throws ApiException {
```

Comment:
this shouldn't throw ApiException it should throw `KinEcosystemException`.
Handle with try and catch and throw the relevant exception through `ErrorUtils.fromApiException`